bentoml/service/env.py

```diff
@@ -367,9 +379,10 @@ def to_dict(self):
                 str(pkg_req) for pkg_req in self._pip_packages.values()
             ]
 
-        env_dict["conda_env"] = self._conda_env._conda_env
+        if self._requirements_txt_file:
+            env_dict["requirements_txt"] = self._requirements_txt_file
```

Comment:
as we discussed earlier about this metadata field, it should probably read the file content here?